src/main/java/com/starl0stgaming/gregicalitystarbound/api/space/planets/worldgen/WorldGenDetails.java

```diff
@@ -0,0 +1,54 @@
+package com.starl0stgaming.gregicalitystarbound.api.space.planets.worldgen;
+
+import com.google.gson.JsonObject;
+import com.starl0stgaming.gregicalitystarbound.api.util.StringUtil;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.world.biome.Biome;
+
+import java.util.List;
+
+public class WorldGenDetails {
+
+    public String name;
+    public int averageGroundLevel;
+    public List<PlanetBiome> biomeList;
+    public IBlockState stone;
+    public IBlockState bedrock;
```

Comment:
GSON needs these to be strings to load them from json, just renaming these to "stoneBlockState" and "bedrockBlockState" and adding 2 string variables with the names "stone" and "bedrock" will be enough